src/org/traccar/database/CommandsManager.java

```diff
@@ -70,6 +73,8 @@ public boolean sendCommand(Command command) throws Exception {
             ActiveDevice activeDevice = Context.getConnectionManager().getActiveDevice(deviceId);
             if (activeDevice != null) {
                 activeDevice.sendCommand(command);
+            } else if (!queueing) {
+                throw new RuntimeException("Commands queueing is not enabled");
```

Comment:
Maybe we should just return something like "Device is offline"?

Reply:
Yes, but a bit more correct "not online" it is unknown or offline. And like it was before:
```java
throw new RuntimeException("Device is not online");
```